### PR TITLE
store raw symbols in decoder::Table

### PR DESCRIPTION
**Pull request by japaric**
the raw symbols are used to check if a symbol is "live" when extracting location information from dwarf
fixes location information in PR #175

### CODE REVIEW

File: decoder/src/lib.rs [hunk 1]
```diff
@@ -113,8 +133,9 @@ impl Table {
         self.entries.is_empty()
     }
 
+    /// Iterates over the raw symbols of the table entries
     pub fn symbols<'s>(&'s self) -> impl Iterator<Item = &'s str> + 's {
```
Comment (japaric) [inline decoder/src/lib.rs:137]: this method relies on `Item` being a 'raw symbol' (as it appears in the `.symtab`)

**Review by jonas-schievink — APPROVED** [review]
Nice!

File: decoder/src/lib.rs (hunk 1 above)
Comment (jonas-schievink) [reply decoder/src/lib.rs:137]: We could rename it to `raw_symbols` if that doesn't cause too much churn, to make it a bit clearer

Comment (japaric) [reply decoder/src/lib.rs:137]: sure!

**Pull request merged**